src/app/components/media/CreateRelatedMediaDialog.js

```diff
@@ -97,7 +97,7 @@ class CreateRelatedMediaDialog extends React.Component {
         </DialogContent>
         <DialogActions>
           <Button id="create-media-dialog__dismiss-button" onClick={this.props.onDismiss}>
-            {this.props.intl.formatMessage(globalStrings.cancel)}
+            {<FormattedMessage {...globalStrings.cancel} />}
```

Comment:
Please remove outer curly braces as they're unnecessary here

Reply:
please check once I add it but what will be the default message I wrote defaultMessage="props".